server/src/domain/sql_backend_handler.rs

```diff
@@ -92,9 +94,16 @@ pub mod tests {
                 user_id: UserId::new(name),
                 email: format!("{}@bob.bob", name).into(),
                 display_name: Some("display ".to_string() + name),
-                first_name: Some("first ".to_string() + name),
-                last_name: Some("last ".to_string() + name),
-                ..Default::default()
+                attributes: vec![
+                    DomainAttributeValue {
+                        name: AttributeName::from("first_name"),
```

Comment:
Why the different style to construct the attribute name? If .into() is enough, let's use that

Reply:
Yeah, that's a bit weird, into() it is.